libraries/Microsoft.Bot.Builder.Runtime/Directory.Build.props

```diff
@@ -0,0 +1,29 @@
+<!-- Contains common properties that apply to projects under the Libraries folder -->
+<Project>
+  <PropertyGroup>
```

Comment:
This is temporary.  We have a P0 issue to resolve missing comments in R12.  This entire file will go away since all it does is disable that warning (error).